src/components/document/sorted-documents.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import { DocumentContextReact } from "./document-context";
+import classNames from "classnames";
+import ArrowIcon from "../../assets/icons/arrow/arrow.svg";
+import { SortedDocument } from "../../models/stores/sorted-documents";
+import { DocumentModelType, getDocumentContext } from "../../models/document/document";
+import { DecoratedDocumentThumbnailItem } from "../thumbnail/decorated-document-thumbnail-item";
+import { useStores } from "../../hooks/use-stores";
+import { logDocumentViewEvent } from "../../models/document/log-document-event";
+import { ENavTab } from "../../models/view/nav-tabs";
+
+import "./sort-work-view.scss";
+
+interface IProps {
+  idx: number;
+  sortedSection: SortedDocument
+}
+
+export const SortedDocuments: React.FC<IProps> = observer(function SortedDocuments(props: IProps) {
+  const { idx, sortedSection } = props;
+  const { persistentUI } = useStores();
+
+  const [showDocuments, setShowDocuments] = useState(false);
+
+  const handleSelectDocument = (document: DocumentModelType) => {
+    persistentUI.openSubTabDocument(ENavTab.kSortWork, ENavTab.kSortWork, document.key);
+    logDocumentViewEvent(document);
+  };
+
+  const handleArrowClick = () => {
+    setShowDocuments(!showDocuments);
```

Comment:
Should this also use persistentUI?

Reply:
I'm not sure either. I don't think the PT story says anything about whether an opened section needs to persist as open. But then it seems like an opened document persists as opened, and you could argue that the opened document's section should always be expanded unless it's closed by the user... I would ask Leslie or Michael about that. Or you could just leave things as is for now and ask about that when you deliver the story.

Reply:
Also, could there be a more meaningful name for this function? `handleArrowClick` is very generic. Maybe something more like `toggleSection` or `handleToggleSection`?